src/prettierRule.ts

```diff
@@ -7,10 +7,21 @@ import * as ts from 'typescript';
 
 export class Rule extends tslint.Rules.AbstractRule {
   public apply(sourceFile: ts.SourceFile): tslint.RuleFailure[] {
+    if (this.sourceFileIsIgnored(sourceFile, this.ruleArguments)) {
+      return [];
+    }
     return this.applyWithWalker(
       new Walker(sourceFile, this.ruleName, this.ruleArguments),
     );
   }
+  private sourceFileIsIgnored(sf: ts.SourceFile, ruleArguments: any[]) {
```

Comment:
```suggestion
  private sourceFileIsIgnored(sourceFile: ts.SourceFile, ruleArguments: any[]) {
```

Full name is preferred over abbr in this project.

Reply:
no problem will change